views/assets/js/chart/main.js

```diff
@@ -7,7 +7,7 @@ for (const chartElem of charts) {
 	const chartOptionsTag = chartElem.querySelector("[data-chart='settings']");
 	const chartCanvasTag = chartElem.querySelector("[data-chart='canvas']");
 	if (chartOptionsTag != null && chartCanvasTag != null) {
-		chartElem.classList.remove("govuk-visually-hidden")
+		chartElem.classList.remove("hidden")
```

Comment:
This seems to somehow fail to remove the `hidden` class from the element.

![](https://github.com/Civil-Service-Human-Resources/lpg-management/assets/97103826/fb123730-b2ef-4240-8acf-600cc983bf27)

![image](https://github.com/Civil-Service-Human-Resources/lpg-management/assets/97103826/ca5bd2df-f2f1-4103-8b9a-364ca77340b4)

Reply:
Tested on Firefox, Edge and Chrome and it's working as expected. Do you maybe have some settings in Safari enabled? Have you cleared your cache/rebuilt the JS etc.

Reply:
Nevermind. Forgot I had to rerun `npm run build` before. All good!